test/server.mjs

```diff
@@ -219,18 +227,17 @@ async function expectNoError(api, key) {
   }
 }
 
-module.exports = { launchProxy, app, disconnectProxy, fetchRequest, expectNoRequest, expectNoError };
-
-if (require.main === module) {
-  const a = app();
-  if (process.argv.length <= 2) {
-    // Launch behind ngrok.
-    launchProxy(a).then(url => console.log('🚆', url));
-  } else {
-    // Launch on the given port.
-    const port = parseInt(process.argv[2], 10);
-    if (isNaN(port)) throw new Error('invalid port');
-    a.listen(port, 'localhost');
-    console.log('http://localhost:' + port);
-  }
-}
+// TODO https://stackoverflow.com/questions/57838022/detect-whether-es-module-is-run-from-command-line-in-node
```

Comment:
Is this still needed?

Reply:
Reinstated in 73077d09d1b4ca66c596a611006e300d19f607d3